packages/manager/apps/catalog-revamp/src/utils/utils.ts

```diff
@@ -0,0 +1,81 @@
+import { groupBy, includes, isEmpty, map, uniq } from 'lodash';
+
+export type Product = {
+  id: number;
+  name: string;
+  description: string;
+  lang: string;
+  categories: string[];
+  url: string;
+  regionTags: string[];
+  productName: string;
+  order: string;
+  universe: string;
+  category: string;
+};
+
+type ProductKey = 'universe' | 'category' | 'name';
+
+export const groupItems = (itemsToGroup: Product[]) => {
```

Comment:
Is this function really useful ?
You should rename it groupItemsByUniverse since it is what it does

Reply:
This function belong to the original Angular code 
packages/manager/apps/hub/src/dashboard/catalog/catalog.controller.js

Reply:
That was not my point.